e2etest/pkg/frr/bgp.go

```diff
@@ -124,3 +124,24 @@ func ContainsCommunity(exec executor.Executor, community string) error {
 	}
 	return nil
 }
+
+// RoutesMatchedLocalPref check if routes match specific local preference value.
+func RoutesMatchedLocalPref(exec executor.Executor, v4LocalPref, v6LocalPref uint32) error {
```

Comment:
We always check only one at the time, I'd pass here the expected localpref and the ip family.

Reply:
not true in case of dual-stack ?

Reply:
But the tests that we are adding here are using this one either set v4localpref or v6localpref I think

Reply:
I will use one value for CI

Reply:
So, can you change this function to accept only one value and the family of that value?